apps/passport-server/src/services/generic-issuance/genericIssuanceService.ts

```diff
@@ -992,8 +1002,8 @@ export class GenericIssuanceService {
       );
       await this.atomDB.clear(validatedNewDefinition.id);
       // purposely not awaited
-      this.restartPipeline(validatedNewDefinition.id);
-      return validatedNewDefinition;
+      const restartPromise = this.restartPipeline(validatedNewDefinition.id);
+      return { definition: validatedNewDefinition, restartPromise };
```

Comment:
This feels a bit ugly, but I needed to be able to await on the pipeline restart in a test in which the pipeline definition gets updated, and this seemed like the simplest way to do it.